config/routes.rb

```diff
@@ -142,10 +142,11 @@
       get   "/settings" => "communities#settings",        as: :settings
       patch "/settings" => "communities#update_settings", as: :update_settings
 
+      get "getting_started_guide(/*all)" => "getting_started_guide#index", as: :getting_started_guide
```

Comment:
They are in different namespace (:admin vs. :styleguide) - isn't that enough?
It would actually be more convenient that they have the same name.